python/lsst/meas/photoz/algorithms/tests/utils.py

```diff
@@ -130,15 +133,15 @@ def do_pz_task(
             os.path.join("${TESTDATA_RAIL_DIR}", model_file),
         )
     )
-    pz_model = PZModel.read(modelpath)
+    photoz_model = PhotozModel.read(modelpath)
     task_config = estimator_class.ConfigClass()
     if config_callback:
         config_callback(task_config)
     task = estimator_class(True, config=task_config)
     to_delete = []
-    output = task.run(pz_model, data)
+    output = task.run(photoz_model, data)
```

Comment:
Even if you don't make the signature to be keyword-only, could you pass these as keyword arguments?